CHANGELOG.md

```diff
@@ -46,6 +62,14 @@ The category for changes related to documentation, testing and tooling. Also, fo
 
     *Manuel Puyol*
 
+* Fix components not rendering in Storybook because of kebab case arguments.
```

Comment:
I made those two small changes as I encounter them during the creation of that PR.

Is it OK to add it to that PR or should I make separates ones?